enzyme/Enzyme/TypeAnalysis/TypeAnalysis.cpp

```diff
@@ -4051,7 +4057,9 @@ TypeResults TypeAnalysis::analyzeFunction(const FnTypeInfo &fn) {
   }
 
   analysis.prepareArgs();
+  analysis.considerRustDebugInfo();
```

Comment:
I'd consider wrapping this in an optional argument -- I think there's already a rust specific one even.

Reply:
Hmm, what do you mean by "wrapping this in an optional argument"? Wrap what to where?